sdk/communication/azure-communication-callautomation/src/main/java/com/azure/communication/callautomation/CallAutomationEventParser.java

```diff
@@ -151,6 +152,8 @@ private static CallAutomationEventBase parseSingleCloudEvent(String data, String
                 ret = mapper.convertValue(eventData, CancelAddParticipantFailed.class);
             } else if (Objects.equals(eventType, "Microsoft.Communication.ConnectFailed")) {
                 ret = mapper.convertValue(eventData, ConnectFailed.class);
+            } else if (Objects.equals(eventType, "Microsoft.Communication.CreateCallFailed")) {
+                ret = mapper.convertValue(eventData, CreateCallFailed.class);                
```

Comment:
Good thing that older library versions will just ignore the returned `null` value and not add them to the result list. Otherwise, this would have been breaking.